README.md

```diff
@@ -29,6 +29,13 @@ Theia is an extensible platform to develop full-fledged multi-language Cloud & D
 - Provide support for multiple languages via the language and debug server protocols
 - Provide modern GUI with javascript UI libraries
 
+
+## Prerequisites
```

Comment:
it is already covered here: https://github.com/theia-ide/theia/blob/master/doc/Developing.md#prerequisites